### DISCUSSION TITLE
PNG to JXL and AVIF

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7135] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

How do I losslessly convert png files to jxl files? Also, when I use the default command for avif files, the output was lossy. How do I fix that?

**Comment by snibgo** [comment]
> How do I losslessly convert png files to jxl files?

```
magick in.png -quality 100 out.jxl
```

> Also, when I use the default command for avif files, the output was lossy. How do I fix that?

As far as I know, IM will always write lossily to AViF.
